pagefaultgames/pokerogue
# [Localization] Add localization about message overwrite the data in selected slot

## What are the changes?

Add localization about message "overwrite the data in selected slot"

## Why am I doing these changes?



There is a need to translate it into Korean.

## What did change?


- Added save-slot-select-ui-handler.ts into each localization folders
- Added "saveSlotSelectorUiHandler" resource into i18next.resources
- Added "saveSlotSelectUiHandler" into each localizatoin config files
- Translated "Overwrite the data in the selected slot?" to Korean
> Other languages remains with English

### Screenshots/Videos


<img width="910" alt="image" src="https://github.com/pagefaultgames/pokerogue/assets/13056250/bd1fc25b-2398-4d06-a7cf-f6f8bebfdfe6">

## How to test the changes?




1. Check out this PR
2. Change language to Korean
3. You can see these changes when you save game data after selecting all of your starter Pokémon.

## Checklist
